streams/id.m3u

```diff
@@ -314,6 +314,8 @@ http://210.210.155.37/uq2663/h/h143/index.m3u8
 http://122.248.43.138:1935/ch16/myStream/playlist.m3u8
 #EXTINF:-1 tvg-id="PalembangTV.id" status="online",Pal TV (720p) [Not 24/7]
 https://v3.siar.us/paltv/live/playlist.m3u8
+#EXTINF:-1 tvg-id="PalembangTV.id" status="online",Pal TV (720p) [Not 24/7]
+http://122.248.43.138:1935/ch7/myStream/playlist.m3u8
```

Comment:
I suggest you play with the number next to the "ch". You might have some surprises.

Reply:
Played with it some time ago. Nothing special with it and some streams in this IP have inconsistent uptime.

Extra info: These streams from this IP also supports mpd (broken), rtmp, and rtsp format.